aibolit/extract_method_baseline/clustering.py

```diff
@@ -0,0 +1,41 @@
+from clustering_test import example  # type: ignore
+from collections import Counter
+from typing import List
+from collections import OrderedDict
+
+
+def check_is_common(dict_file, statement_1: int, statement_2: int) -> bool:
```

Comment:
Provide proper types, check the output of `_extract_method_statements_semantic` method.
And we better peek a more proper name.

Reply:
Why you don't like those names?